pixi.toml

```diff
@@ -55,17 +55,20 @@ finch-tensor = ">=0.1.31"
 SPARSE_BACKEND = "Finch"
 
 [feature.finch.target.osx-arm64.activation.env]
+SPARSE_BACKEND = "Finch"
 PYTHONFAULTHANDLER = "${HOME}/faulthandler.log"
 
 [feature.mlir.dependencies]
 scipy = ">=0.19"
-mlir-python-bindings = "19.*"
+
+[feature.mlir.target.osx-arm64.pypi-dependencies]
+finch-mlir = ">=0.0.2"
```

Comment:
Nitpick: I think we already want to use `finch-mlir` for all OSes/architectures (doesn't need to be in this PR, I'm not using pixi).

Reply:
There seems to be a bug in the `pixi` solver that prevents the installation on Linux.